src/components/akoma-ntoso/akoma-ntoso.tsx

```diff
@@ -1,10 +1,72 @@
-import { Component, h } from '@stencil/core';
+import { Component, h, Prop, Watch } from '@stencil/core';
 
 @Component({
   tag: 'la-akoma-ntoso',
   styleUrl: 'akoma-ntoso.scss',
 })
 export class AkomaNtoso {
+  /**
+   * String of standard parsed into thee follwoing metadata properties frbr-attributes:
+   * frbrCountry, frbrType, frbrSubtype, frbrAuthor, frbrDate, frbrNumber, frbrExpressionDate,
+   * frbrLanguage
+   * */
+  @Prop({ reflect: true, mutable: true}) frbrExpressionUri?: string;
+  /**
+   * A metadata property containing a ISO 3166-1 Alpha-2 code for the country or jurisdiction
```

Comment:
@longhotsummer is there a good resource to reference to get good frbr prop definitions

Reply:
There's the AKN naming convention standard https://docs.oasis-open.org/legaldocml/akn-nc/v1.0/akn-nc-v1.0.html or our AKN4Africa recommendation https://laws.africa/publications/akn4africa-recommendation.html